### DISCUSSION TITLE
How to configure clients dynamically

### DISCUSSION THREAD

**Discussion by fit4s** [spawnia/sailor#72] [Q&A]
Hello. 

According to the README.md under the heading "Dynamic clients" I can use a method with the name "setClient" to configure clients dynamically. After I successfully generated PHP code with vendor/bin/sailor, the method is not defined and not inherited from the parent class. Is there anything to consider in the configuration?

I am very grateful for any help.
Thanks.

**Comment by spawnia** [comment]
I don't see the problem, the method exists on the parent class `Operation`: https://github.com/spawnia/sailor/blob/dd556fd554707dc25ec308875b663ffcf96669cd/src/Operation.php#L128-L131

**Comment by fit4s** [comment]
Oh, sorry, my mistake
Composer has installed an old version (0.1.0).
That's why the method was not present.
Thanks for the fast reply.
